# Input topics handled/muxed.
# For each topic:
# - name    : name identifier to select the topic
# - topic   : input topic of geometry_msgs::Twist type
# - timeout : timeout in seconds to start discarding old messages, and use 0.0 speed instead
# - priority: priority in the range [0, 255]; the higher the more priority over other topics

command_inputs:
-
  sub_topic : /cmd_vel/joystick
  pub_topic : /managed/joy
  timeout   : 0.5
  stamped   : true
-
  sub_topic: key
  pub_topic: /managed/key
  timeout: 0.5
  stamped: false
-
  sub_topic: test
  pub_topic: /managed/test
  timeout: 0.5
  stamped: false

